products/green_fast_track/seeds/_seeds.yml

```diff
@@ -32,3 +33,36 @@ seeds:
           - not_null
           - accepted_values:
               values: ["TRUE", "FALSE"]
+
+  - name: expected_green_fast_track_pilot_projects
+    description: Mock pilot projects and their expected outputs
+    config:
+      # empty columns default to integer type
```

Comment:
I'd rather not have to declare these column types (other than `bbl` and `geom`), but I couldn't find any other way

csv values that properly end up as `value | NULL | value` in the table that I tried are `value,,value`, `value,"",value`, and `value,NULL,value`. but in all cases the inferred column type is integer when all values in the column are `NULL`. when that happens, the equality test fails because the "actual" data's column is type text

Reply:
Makes sense, good to know